tests/labels/test_api.py

```diff
@@ -232,6 +249,16 @@ async def test_remove(error, spawn_client, pg: AsyncEngine, resp_is):
 
     await resp_is.no_content(resp)
 
+    responses = [await client.get(f"/samples/{sample['_id']}") for sample in mock_samples]
+
+    samples = [await response.json() for response in responses]
+
+    label_ids = [label["id"]
+                 for sample in samples
+                 for label in sample["labels"]]
+
+    assert 1 not in label_ids
```

Comment:
Did you setup Black as previously discussed? This formatting is not compliant.

```
sudo python3 -m pip install black
```
Then follow the file watcher guide for Pycharm:
https://black.readthedocs.io/en/stable/integrations/editors.html#pycharm-intellij-idea